blocks/inspector-controls/checkbox-control/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * WordPress dependencies
+ */
+import { withInstanceId } from 'components';
+
+/**
+ * Internal dependencies
+ */
+import BaseControl from './../base-control';
+import './style.scss';
+
+function CheckboxControl( { label, heading, checked, instanceId, onChange, ...props } ) {
+	const id = 'inspector-checkbox-control-' + instanceId;
+	const onChangeValue = ( event ) => onChange( event.target.value );
+
+	return (
+		<BaseControl label={ heading } id={ id }>
+			<label className="blocks-checkbox-control__label" htmlFor={ id }>
```

Comment:
Well! According to the [HTML Documentation](https://www.w3.org/TR/html401/interact/forms.html#h-17.9.1), we can have two labels for the same input. So, let's leave it as is

> More than one LABEL may be associated with the same control by creating multiple references via the for attribute.